crypto/fipsmodule/bn/rsaz_exp.h

```diff
@@ -100,6 +100,24 @@ void rsaz_1024_gather5_avx2(BN_ULONG val[40], const BN_ULONG tbl[32 * 18],
 // |bn_reduce_once|.
 void rsaz_1024_red2norm_avx2(BN_ULONG norm[16], const BN_ULONG red[40]);
 
+#if !defined(MY_ASSEMBLER_IS_TOO_OLD_FOR_512AVX)
```

Comment:
no need for this `ifdef`, see line 28

Reply:
This one is 512AVX which is distinct from the one at line 28, which is just AVX.

Reply:
That makes sense, but is there not also a case where we want to _include_ the AVX2 functions above, but _exclude_ the AVX512 functions below because of an assembler which is aware of AVX2 but not AVX512?